libiocage/lib/Jail.py

```diff
@@ -601,7 +797,14 @@ def __getattr__(self, key):
 
         raise AttributeError(f"Jail property {key} not found")
 
-    def getattr_str(self, key):
+    def getstring(self, key):
```

Comment:
i wonder if that's an improvement in any regard, esp for downstream users who consume our output automatically.